src/components/MediaList/MediaList.tsx

```diff
@@ -202,7 +220,10 @@ const ItemText = ({ item }: { item: ItemType }) => {
 
   return (
     <Text textStyle="body.medium">
-      {item.label}: <HandleLink url={item.description}>{item.description}</HandleLink>
+      {item.label}:{" "}
```

Comment:
Er ikkje fan av whitespace på denne måten. Kan du heller gjøre nokke sånt:
```suggestion
      {`${item.label}: `}
```

Reply:
Merket ikke dette. Irriterende at siden koden blir for lang så blir linjen formatert på denne måten også slenger vscode inn en sånn whitespace 🙄